docs/content/providers/overview.md

```diff
@@ -35,9 +35,10 @@ Below is the list of the currently supported providers in Traefik.
 | [Rancher](./rancher.md)               | Orchestrator | Label                      |
 | [File](./file.md)                     | Manual       | TOML/YAML format           |
 | [Consul](./consul.md)                 | KV           | KV                         |
-| [etcd](./etcd.md)                     | KV           | KV                         |
+| [Etcd](./etcd.md)                     | KV           | KV                         |
 | [Redis](./redis.md)                   | KV           | KV                         |
 | [ZooKeeper](./zookeeper.md)           | KV           | KV                         |
+| [Http](./http.md)                     | Manual       | JSON format                |
```

Comment:
https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol

NIT: I noticed we use `Http` and `HTTP` interchangeably throughout the documentation and I recommend the naming standard as noted in the RFC should be used for consistency.